### PR TITLE
Btw enterprise update 030218

### PR DESCRIPTION
**Pull request by btwisted**
This PR covers all the topics-new and existing- and json.map modifications under Enterprise.

### CODE REVIEW

File: v6/enterprise/intro_to_enterprise.md [hunk 1]
```diff
@@ -6,23 +6,29 @@ tags:
 warning: false
 ---
 
-Postman’s vision is to help you build a super fast and smooth workflow for API development keeping some key questions in mind:
+Postman helps Enterprise users build a fast, smooth API development workflow that monitors the health and status of APIs, produces quality documentation, connects to third party tools to enhance the API development workflow, and implements an agile process to test APIs.
```
Comment (claireriley) [inline v6/enterprise/intro_to_enterprise.md]: This seems kind of like multiple sentences smashed into one. Maybe could simplify?

This is in our one-pager:
> Postman Enterprise supports every aspect of API development, including design & mock, debugging, automated testing, documentation, monitoring, and publishing.

Maybe something similar to that? I don't know if we want to emphasize the differences between enterprise and pro at the start or just later on in the doc?

File: v6/enterprise/intro_to_enterprise.md [hunk 2]
```diff
@@ -6,23 +6,29 @@ tags:
...
 
-*   Where are my APIs and do I know how they are changing?
-*   Are they documented properly?
-*   How do I follow an agile process to test them?
-*   Are they working properly all the time?
-*   Can I connect all my tools with my API development workflow?
+Postman provides Enterprise users with:
 
-Enterprise includes all the advanced capability of Postman Pro, plus features available only to Enterprise customers with solutions that satisfy each of the questions above. 
+* Single Sign-On (SSO): Access to the the company’s Postman instance by using the customer’s existing identity management solution. Postman Enterprise currently supports multiple SSO providers – Okta, OneLogin, Duo, Ping Identity, AD FS, and GSuite. Postman provides SAML 2.0-compliant identity provider support.
```
Comment (claireriley) [inline v6/enterprise/intro_to_enterprise.md]: tried to rephrase a bit - I think "access to the company's Postman instance" could be misconstrued

> Single Sign-On (SSO): Access your Postman team by using your company's pre-existing identity management solution. Postman Enterprise currently supports all SAML 2.0-compliant identity providers, including popular choices like Okta, OneLogin, Duo, Ping Identity, AD FS, and GSuite.

File: v6/enterprise/intro_to_enterprise.md [hunk 3]
```diff
@@ -6,23 +6,29 @@ tags:
...
 
-*   Team [collaboration](/docs/postman/team_library/sharing) for the single source of truth about your API, or review historical versions and the latest updates.
-*   API [documentation](/docs/postman/api_documentation/intro_to_api_documentation) to share public or private documentation, beautifully viewable via a web page.
-*   Powerful [mock servers](/docs/postman/mock_servers) to simulate the real API and decouple teams.
-*   Collection [monitoring](/docs/postman/monitors/intro_monitors) to check for the performance, uptime and correctness of your API.
-*   [Out-of-the-box integrations](/docs/pro/integrations/intro_integrations) and the [Postman API](/docs/pro/pro_api/intro_api) to connect your different tools.
-*   [Dedicated IP addresses](/docs/postman/monitors/intro_monitors#monitoring-resources-in-multiple-regions) for API monitoring.
+* Audit logs: Ability to track activities within teams, particularly when collaborating on code. With Audit Logs, Postman Enterprise users can review every change, see who made it and when it was made in a user-friendly dashboard.
```
Comment (claireriley) [inline v6/enterprise/intro_to_enterprise.md]: I think we should double check on this one as I'm unsure if everything is in the "Audit Logs" web view or if some changes are only logged in the activity feed - sync isn't working for me right now, but noting here to remember to do so (w/ caninecuteness team) pre-publication.

File: v6/enterprise/intro_to_enterprise.md [hunk 4]
```diff
@@ -6,23 +6,29 @@ tags:
...
-*   API [documentation](/docs/postman/api_documentation/intro_to_api_documentation) to share public or private documentation, beautifully viewable via a web page.
-*   Powerful [mock servers](/docs/postman/mock_servers) to simulate the real API and decouple teams.
-*   Collection [monitoring](/docs/postman/monitors/intro_monitors) to check for the performance, uptime and correctness of your API.
-*   [Out-of-the-box integrations](/docs/pro/integrations/intro_integrations) and the [Postman API](/docs/pro/pro_api/intro_api) to connect your different tools.
-*   [Dedicated IP addresses](/docs/postman/monitors/intro_monitors#monitoring-resources-in-multiple-regions) for API monitoring.
+* Audit logs: Ability to track activities within teams, particularly when collaborating on code. With Audit Logs, Postman Enterprise users can review every change, see who made it and when it was made in a user-friendly dashboard. 
+
+* Extended support and billing: Dedicated support, onboarding and billing options for larger organizations with more structured policies around procurement, provisioning and ongoing customer success
```
Comment (claireriley) [inline v6/enterprise/intro_to_enterprise.md]: I vote oxford commas x2

File: v6/enterprise/intro_to_enterprise.md (hunk 3 above)
Comment (claireriley) [reply v6/enterprise/intro_to_enterprise.md]: also would un-capitalize the second "Audit Logs" as it doesn't fit with the formatting of the rest of the doc

File: v6/enterprise/intro_to_enterprise.md [hunk 5]
```diff
@@ -6,23 +6,29 @@ tags:
...
+
+* Team [collaboration](/docs/v6/postman/workspaces/creating_workspaces){:target="_blank"} for the single source of truth about your API.  
+
+* API [documentation](/docs/postman/api_documentation/intro_to_api_documentation){:target="_blank"} to share public or private documentation in a web page.
+
+* [Mock servers](/docs/postman/mock_servers){:target="_blank"} to simulate the real API and decouple teams collection [monitoring](/docs/postman/monitors/intro_monitors) to check the performance of your API.
+
+* [Out-of-the-box integrations](/docs/pro/integrations/intro_integrations){:target="_blank"} and the [Postman API](/docs/pro/pro_api/intro_api){:target="_blank"} to connect your different tools.
+
+* [Dedicated IP addresses](/docs/postman/monitors/intro_monitors#monitoring-resources-in-multiple-regions){:target="_blank"} for API monitoring.
```
Comment (claireriley) [inline v6/enterprise/intro_to_enterprise.md]: this one is only for Enterprise, so we'll want to move this up to the first list

File: v6/enterprise/intro_to_enterprise.md [hunk 6]
```diff
@@ -6,23 +6,29 @@ tags:
...
-*   Collection [monitoring](/docs/postman/monitors/intro_monitors) to check for the performance, uptime and correctness of your API.
-*   [Out-of-the-box integrations](/docs/pro/integrations/intro_integrations) and the [Postman API](/docs/pro/pro_api/intro_api) to connect your different tools.
-*   [Dedicated IP addresses](/docs/postman/monitors/intro_monitors#monitoring-resources-in-multiple-regions) for API monitoring.
+* Audit logs: Ability to track activities within teams, particularly when collaborating on code. With Audit Logs, Postman Enterprise users can review every change, see who made it and when it was made in a user-friendly dashboard. 
+
+* Extended support and billing: Dedicated support, onboarding and billing options for larger organizations with more structured policies around procurement, provisioning and ongoing customer success
+
+In addition to SSO, audit logs, and extended support and billing, the Enterprise plan shares Postman Pro features, such as:
+
+* Team [collaboration](/docs/v6/postman/workspaces/creating_workspaces){:target="_blank"} for the single source of truth about your API.
```
Comment (claireriley) [inline v6/enterprise/intro_to_enterprise.md]: Possible rewording:
> Team [collaboration](/docs/v6/postman/workspaces/creating_workspaces){:target="_blank"} to provide a single source of truth for your APIs.

File: v6/enterprise/intro_to_enterprise.md [hunk 7]
```diff
@@ -6,23 +6,29 @@ tags:
...
+
+* Extended support and billing: Dedicated support, onboarding and billing options for larger organizations with more structured policies around procurement, provisioning and ongoing customer success
+
+In addition to SSO, audit logs, and extended support and billing, the Enterprise plan shares Postman Pro features, such as:
+
+* Team [collaboration](/docs/v6/postman/workspaces/creating_workspaces){:target="_blank"} for the single source of truth about your API.  
+
+* API [documentation](/docs/postman/api_documentation/intro_to_api_documentation){:target="_blank"} to share public or private documentation in a web page.
+
+* [Mock servers](/docs/postman/mock_servers){:target="_blank"} to simulate the real API and decouple teams collection [monitoring](/docs/postman/monitors/intro_monitors) to check the performance of your API.
```
Comment (claireriley) [inline v6/enterprise/intro_to_enterprise.md]: Is monitoring meant to be a second bullet point? I might add to mocks that decoupling allows different members of your development team to work in parallel (or in tandem(?)) rather than having to wait on different pieces to be built out before moving forward. For monitoring, I would add something suggesting "automating" as the phrasing kind of sounds like a one-off action atm.

File: v6/enterprise/intro_to_enterprise.md [hunk 8]
```diff
@@ -6,23 +6,29 @@ tags:
...
+
+* [Out-of-the-box integrations](/docs/pro/integrations/intro_integrations){:target="_blank"} and the [Postman API](/docs/pro/pro_api/intro_api){:target="_blank"} to connect your different tools.
+
+* [Dedicated IP addresses](/docs/postman/monitors/intro_monitors#monitoring-resources-in-multiple-regions){:target="_blank"} for API monitoring.
+
+
+Want to learn more about Postman Enterprise? Contact [Enterprise support](http://pages.getpostman.com/Enterprise-Sales_Contact-Us.html){:target="_blank"}.
```
Comment (claireriley) [inline v6/enterprise/intro_to_enterprise.md]: maybe rephrase to "Contact our Enterprise support team [here]"?

**Comment by claireriley** [comment]
@btwisted ^

File: v6/enterprise/sso/admin_dashboard.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+Note: Only a team admin can configure SSO for a Postman Team.
+
+Note: Only if the team is subscribed to an enterprise plan, the option to configure SSO will be displayed.
+
+Steps for setting up SSO:
+
+Click the Team dropdown in the navigation bar. Look for Team Settings in the list items and click it.
+
+IMAGE
```
Comment (loopDelicious) [inline v6/enterprise/sso/admin_dashboard.md]: Are these placeholders? 🍰  

@btwisted
